include/seqan3/alignment/pairwise/alignment_range.hpp

```diff
@@ -253,6 +157,119 @@ class alignment_range
 //!\brief Deduces from the passed alignment_executor_type
 template <typename alignment_executor_type>
 alignment_range(alignment_executor_type &&) -> alignment_range<std::remove_reference_t<alignment_executor_type>>;
-//!}
+//!\}
+
+/*!\brief The iterator of seqan3::detail::alignment_range.
+ * \implements std::input_iterator
```

Comment:
tparam missing

Reply:
It has no tparam. What you see is the template parameter of the encapsulating class. So nothing to document here.